velox/common/caching/AsyncDataCache.h

```diff
@@ -678,4 +707,43 @@ T percentile(Next next, int32_t numSamples, int percent) {
   return values.empty() ? 0 : values[(values.size() * percent) / 100];
 }
 
+// Describes the outcome of readPins().
+struct ReadPinsResult {
```

Comment:
This struct appears to describe statistics about the read operator, not results. The results are written into the CachePins, right? Perhaps, rename to ReadPinsStats.